### PR TITLE
:potato: Ground nut cooking :sweet_potato: 

### PR DESCRIPTION
**Pull request by pjf**
#### Summary

Content "Ground nuts can now be prepared for cooking, and used much like potato"

#### Purpose of change

Ground nuts (which are really tubers, with a nutty taste) were introduced in #54411, and while their description mentioned they were better cooked, no recipes existed for them.

This PR makes them available in cooking recipes, while trying to minimise recipe bloat.

#### Describe the solution

Ground nuts naturally contain protease inhibitors, and are thus mildly toxic raw.

Prepared ground nuts involve peeling, cleaning, and lightly boiling the tubers to deactivate the protease inhibitors and make vegetable more palatable. They can then be used much like potatoes. This is a somewhat labour intensive process until the survivor is proficient at forage cooking, knife skills, and basic food prep.

Added a `veggy_potatolike` group for tubers that can be used like potatoes, and edits recipes to use it where appropriate. This means they can be baked, boiled, or fried into chips. Smoking and drying are also possible. Recipes which accept any type of vegetable will accept prepared ground nuts.

Baked, boiled, fried, and dried ground nuts use conditional naming to produce correctly named items, without any additional recipes.

I've used a ratio of 5 prepared ground nuts = 1 potato equivalent in recipes, as this gives approximately the same nutrition and weight.

I've also got a separate 'boiled ground nuts' recipes that can boil individual tubers, so the player isn't left wondering to do with leftover tubers. It was the first recipe I wrote in the PR, but I'm not convinced it's needed. Survivors can just eat the prepared ground nuts (hot) for a mediocre meal.

#### Describe alternatives you've considered

Increasing the nutrition of individual ground nuts and reducing their spawn, to make them 1:1 equivalent to potatoes.

Removing the "boiled ground nuts" recipe (very willing to do this, feedback appreciated)

#### Testing

Loaded my innawood save. Prepared some ground nuts. Baked them into baked ground nuts, yum! Dried them into dried veggie, ground nuts. Poked around the recipe interface and inspected items to make sure they looked good.

#### Additional context

I've only just realised that I don't know if we have standard cooking and fuel requirements, so mine might be off. Please feel free to adjust.

I live in Australia, where ground nuts are not available, and thus I've had to rely upon [YouTube Videos](https://www.youtube.com/watch?v=s2c9451Zxm8), rather than local knowledge and going out to the woods myself for research.

This PR also adjusts cooked potatoes to `looks_like` potatoes, and remove what appeared to be a misplaced nutrient override on baked potatoes.

Each commit in this PR is designed to be understandable and easy to review by itself, to make the review process easier.

Happy forager feasting!

### CODE REVIEW

**Comment by pjf** [comment]
Tagging in @NetSysFire for comment, as the original author of the ground nut content. ♥

**Comment by pjf** [comment]
Test failures seem to be about lab post-its, which were not harmed in the making of this pull request.

File: data/json/items/comestibles/junkfood.json [hunk 1]
```diff
@@ -675,9 +675,10 @@
     "type": "COMESTIBLE",
     "id": "fresh_fries",
     "name": { "str_sp": "French fries" },
+    "conditional_names": [ { "type": "COMPONENT_ID", "condition": "groundnut_prepared", "name": { "str_sp": "ground nut fries" } } ],
     "copy-from": "fries",
     "spoils_in": "3 days",
-    "description": "Deep-fried potatoes with a touch of salt.  Crunchy and delicious.",
+    "description": "Deep-fried potatoes or other tuber, with a touch of salt.  Crunchy and delicious.",
```
Comment (Cimanyd0) [inline data/json/items/comestibles/junkfood.json:681]:
```suggestion
    "description": "Deep-fried potatoes or other tubers, with a touch of salt.  Crunchy and delicious.",
```

File: data/json/items/comestibles/veggy_dishes.json [hunk 2]
```diff
@@ -1271,5 +1280,26 @@
...
     "fun": 1,
     "vitamins": [ [ "vitC", 32 ], [ "calcium", 4 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "groundnut_prepared",
+    "name": { "str_sp": "prepared ground nuts" },
+    "copy-from": "groundnut",
+    "spoils_in": "2 days",
+    "description": "Ground nuts that have been peeled and then boiled, to destroy the harmful protease inhibitors.  Safe to eat, but needs further cooking to become palatable. Can be used much like potato.",
```
Comment (Cimanyd0) [inline data/json/items/comestibles/veggy_dishes.json:1290]:
```suggestion
    "description": "Ground nuts that have been peeled and then boiled, to destroy the harmful protease inhibitors.  Safe to eat, but need further cooking to become palatable. Can be used much like potatoes.",
```

File: data/json/items/comestibles/veggy_dishes.json [hunk 3]
```diff
@@ -1271,5 +1280,26 @@
...
+    "smoking_result": "dry_veggy",
+    "healthy": 0,
+    "fun": -2
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "groundnut_boiled",
+    "name": { "str_sp": "boiled ground nuts" },
+    "copy-from": "groundnut_prepared",
+    "description": "Boiled ground nuts. They look and taste a little like potato, but with a slight nutty taste.",
```
Comment (Cimanyd0) [inline data/json/items/comestibles/veggy_dishes.json:1302]:
```suggestion
    "description": "Boiled ground nuts. They look and taste a little like potatoes, but with a slight nutty taste.",
```

**Review by Cimanyd0 — CHANGES_REQUESTED** [review]
A few grammar suggestions.

File: data/json/items/comestibles/veggy_dishes.json (hunk 3 above)
Comment (pjf) [reply data/json/items/comestibles/veggy_dishes.json]: Thank you! I've applied all these suggestions!

File: data/json/items/comestibles/veggy_dishes.json [hunk 4]
```diff
@@ -1271,5 +1280,26 @@
...
     "fun": 1,
     "vitamins": [ [ "vitC", 32 ], [ "calcium", 4 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "groundnut_prepared",
+    "name": { "str_sp": "prepared ground nuts" },
+    "copy-from": "groundnut",
+    "spoils_in": "2 days",
+    "description": "Ground nuts that have been peeled and then boiled, to destroy the harmful protease inhibitors.  Safe to eat, but need further cooking to become palatable. Can be used much like potatoes.",
```
Comment (Maleclypse) [inline data/json/items/comestibles/veggy_dishes.json:1290]:
```suggestion
    "description": "Ground nuts that have been peeled and then boiled, to destroy the harmful protease inhibitors.  Safe to eat, but need further cooking to become palatable.  Can be used much like potatoes.",
```

File: data/json/items/comestibles/veggy_dishes.json [hunk 5]
```diff
@@ -1271,5 +1280,26 @@
...
+    "smoking_result": "dry_veggy",
+    "healthy": 0,
+    "fun": -2
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "groundnut_boiled",
+    "name": { "str_sp": "boiled ground nuts" },
+    "copy-from": "groundnut_prepared",
+    "description": "Boiled ground nuts. They look and taste a little like potatoes, but with a slight nutty taste.",
```
Comment (Maleclypse) [inline data/json/items/comestibles/veggy_dishes.json:1302]:
```suggestion
    "description": "Boiled ground nuts.  They look and taste a little like potatoes, but with a slight nutty taste.",
```

**Review by NetSysFire — APPROVED** [review]
I like this so far. I have some local unpushed changes for a couple of other recipe changes which conflict with yours but it will be fine.

**Comment by pjf** [comment]
Thanks for the merge!

**Pull request merged**